messages.js

```diff
@@ -144,13 +144,14 @@ exports.embedEventAsImage = async (event, locale) => {
 };
 
 exports.embedInventoryAsImage = async event => {
+  const l = exports.getI18n(locale);
```

Comment:
`locale` is a function parameter that is not being set today in the function parameters.

You can change  line 146 for:
```
exports.embedInventoryAsImage = async (event, locale) => {
```
just like line 122 does, so the function parameter if set accordingly.